charts/ndb-operator/README.md

```diff
@@ -10,6 +10,7 @@ NDB operator supports these functionalities:
 1. [Install](https://portal.nutanix.com/page/documents/details?targetId=Nutanix-NDB-User-Guide-v2_5:top-installation-c.html) NDB 2.5.
 2. [Install](https://helm.sh/docs/intro/install/) Helm v3.0.0.
 3. [Install](https://kubernetes.io/docs/setup/) a Kubernetes cluster.
+4. Installing the cert-manager. Please follow the instructions [here] (https://cert-manager.io/docs/installation/#getting-started). Ensure that the cert-manager resouces are up and running successfully before installing the NDB operator.
```

Comment:
Based on the more recent commits, if you have anything to update in the README, do reach out to me separately.
Eg database description field was added in the spec - that may require us to update the template here etc.

Reply:
There are already some optional fields that we have included in the base template. If we want to be consistent, this can be added as well. 

Additionally I believe there is no other better way to make users aware of such a field being accepted in the DB Spec, other than this. So I'd prefer to have it there.

Reply:
I agree - lets add it.